bindings/python/eblob_python.cpp

```diff
@@ -75,18 +75,17 @@ struct eblob_py_iterator : eblob_iterate_control, boost::python::wrapper<eblob_i
 	static int iterator(struct eblob_disk_control *dc, struct eblob_ram_control *rc __attribute__((unused)),
 			    int fd, uint64_t data_offset, void *priv, void *thread_priv __attribute__((unused)))
 	{
-		std::unique_ptr<char> p(new char[dc->data_size]);
+		std::string p(dc->data_size, 0);
```

Comment:
String c'tor initializes memory, but it isn't required here.
This one doesn't initializes memory: std::unique_ptr<char[]> p(new char[dc->data_size]);